crates/sui-indexer/src/models/transaction_index.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::schema::{input_objects, move_calls, recipients};
+use diesel::prelude::*;
+
+#[derive(Queryable, Insertable, Debug, Clone, Default)]
+#[diesel(table_name = input_objects)]
+pub struct InputObject {
+    pub id: Option<i64>,
+    pub transaction_digest: String,
+    pub checkpoint_sequence_number: i64,
+    pub epoch: i64,
+    pub object_id: String,
+    pub object_version: Option<i64>,
+}
+
+#[derive(Queryable, Insertable, Debug, Clone, Default)]
+#[diesel(table_name = move_calls)]
+pub struct MoveCall {
+    pub id: Option<i64>,
+    pub transaction_digest: String,
+    pub checkpoint_sequence_number: i64,
+    pub epoch: i64,
+    pub sender: String,
+    pub move_package: String,
+    pub move_module: String,
+    pub move_function: String,
+}
+
+#[derive(Queryable, Insertable, Debug, Clone, Default)]
+pub struct Recipient {
```

Comment:
Maybe a dumb question: Should we add `sender` field to this table? Frontend team wants to be able to be able to simultaneously filter(and paginated) on both `FromAddress` and `ToAddress`? More generally, how do we fulfill `MatchAll` for multiple filters on indexer?

Reply:
generally, we would JOIN`transactions` with `recipients` and compose query from there.
When we start supporting query compose, and if we want to optimize this, this can be easily added together with the query compose support.

Reply:
Thanks for the clarification! Filtering on both `FromAddress` and `ToAddress` is a common query(used by explorers and wallets frequently) and may be worth some optimization(e.g., avoid joining). Should we add just add a `sender` field to this table? cc @Jordan-Mysten

Reply:
agreed this is an easy change, I meant the bigger change in on the query definition, whenever we add a query type to support both "from" and "to", adding the sender here is more relevant to that PR